development/Dockerfile

```diff
@@ -1,45 +1,73 @@
+# -------------------------------------------------------------------------------------
+# Nautobot Plugin Development Dockerfile Template
+# Version: 1.0.0
+#
+# Plugins that need to add additional steps or packages can do in the section below.
+# -------------------------------------------------------------------------------------
+# !!! USE CAUTION WHEN MODIFYING LINES BELOW
+
+# Accepts a desired Nautobot version as build argument, default to 1.4.7
 ARG NAUTOBOT_VER="1.4.7"
-ARG PYTHON_VER=3.8
-FROM ghcr.io/nautobot/nautobot-dev:${NAUTOBOT_VER}-py${PYTHON_VER}
 
-ENV prometheus_multiproc_dir=/prom_cache
+# Accepts a desired Python version as build argument, default to 3.7
+ARG PYTHON_VER="3.8"
```

Comment:
There is a mismatch here (and in the previous version of this file):

**Nautobot Version**
- `1.4.7` previous default in Dockerfile
- `latest` current invoke default: https://github.com/nautobot/nautobot-plugin-circuit-maintenance/blob/5bbf9c8763b9f6e255703ca269ba95b2c63df32c/tasks.py#L43
- `1.0.0-beta.4` current minimum in `PluginConfig`: https://github.com/nautobot/nautobot-plugin-circuit-maintenance/blob/5bbf9c8763b9f6e255703ca269ba95b2c63df32c/nautobot_circuit_maintenance/__init__.py#L86 
- `*` current minimum in `pyproject.toml`. https://github.com/nautobot/nautobot-plugin-circuit-maintenance/blob/5bbf9c8763b9f6e255703ca269ba95b2c63df32c/pyproject.toml#L23

**Python Version**
- `3.8` previous default in Dockerfile
- `3.7` current defaults in invoke and `pyproject.toml`

I will go ahead and update these in my PR but I would like to know the maintainer's preferences.

Reply:
I would use Nautobot latest for development, while leaving the all Nautobot versions supported. For Python, I would go with 3.8 for development, and keep testing for 3.7 in the pipeline.
is this in line with your recommendations?